6.0/fpm/Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM php:5.6-fpm
+
+RUN apt-get update && apt-get install -y \
+  bzip2 \
+  libcurl4-openssl-dev \
+  libfreetype6-dev \
+  libicu-dev \
+  libjpeg-dev \
+  libmcrypt-dev \
+  libmemcached-dev \
+  libpng12-dev \
+  libpq-dev \
+  libxml2-dev \
+  && rm -rf /var/lib/apt/lists/*
+
+#gpg key from https://owncloud.org/owncloud.asc
+RUN gpg --keyserver ha.pool.sks-keyservers.net --recv-keys E3036906AD9F30807351FAC32D5D5E97F6978A26
+
+# https://doc.owncloud.org/server/8.1/admin_manual/installation/source_installation.html#prerequisites
+RUN docker-php-ext-configure gd --with-png-dir=/usr --with-jpeg-dir=/usr \
+ && docker-php-ext-install gd intl mbstring mcrypt mysql opcache pdo_mysql pdo_pgsql pgsql zip
```

Comment:
Swap the spaces to tabs, line 21. Applies to 6.0, 7.0, 8.0, and 8.1 Dockerfiles.